src/main/java/clickup/ui/pages/task/TaskPage.java

```diff
@@ -71,6 +88,30 @@ public void clickAddTagButton() {
         tagForm = new TagForm(webDriverManager);
     }
 
+    /**
+     * Clicks a Tag.
+     *
+     * @param tagName tag to be clicked
+     * @author Jorge Caceres
+     */
+    public void clickTag(final String tagName) {
+        tagButton = By.xpath(String.format(tagSettingsButton, tagName));
+        webDriverActions.clickElement(tagButton);
+        tagSettings = new TagSettings(webDriverManager);
```

Comment:
Is not better to return a  TagSettings?

Reply:
No, because it is a component on the taskpage